tox.ini

```diff
@@ -27,6 +28,7 @@ commands =
 [testenv:format]
 # Attempt to auto-resolve lint errors before they are raised.
 # To execute, run `tox -e format`
+allowlist_externals = poetry
```

Comment:
Doesn't this cascade from line 9? I'm not even sure we really need this

Reply:
Yeah actually the real change that was needed is whitelist_externals -> allowlist_externals, this stuff is cruft from my troubleshooting that. Removing